http-server-netty/src/test/groovy/io/micronaut/http/server/netty/cors/CorsFilterSpec.groovy

```diff
@@ -15,16 +15,27 @@
  */
 package io.micronaut.http.server.netty.cors
 
+import io.micronaut.context.ApplicationContext
 import io.micronaut.http.*
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
 import io.micronaut.http.server.HttpServerConfiguration
 import io.micronaut.http.server.cors.CorsFilter
 import io.micronaut.http.server.cors.CorsOriginConfiguration
+import io.micronaut.runtime.server.EmbeddedServer
+import io.micronaut.web.router.RouteMatch
+import io.micronaut.web.router.Router
+import org.apache.http.client.utils.URIBuilder
 import spock.lang.Specification
 
+import java.util.stream.Collectors
+
 import static io.micronaut.http.HttpHeaders.*
 
 class CorsFilterSpec extends Specification {
 
+    EmbeddedServer embeddedServer = ApplicationContext.run(EmbeddedServer)
```

Comment:
construct this in the test methods, and properly shut it down after the test is done